types/babel__core/index.d.ts

```diff
@@ -82,7 +92,7 @@ export interface TransformOptions {
      *
      * Default: env vars
      */
-    envName?: string | null;
+    envName?: string;
```

Comment:
This is outside the scope of what I checked but we probably should get rid of all the other `null`s too unless they are explicitly allowed / handled in the code. Babel uses destructuring to provide a default `envName` (so only `undefined` is handled), and the rest of the code _in babel_ also assumes it will be a `string` once defaulted.